.travis.yml

```diff
@@ -5,7 +5,7 @@ dist: trusty
 # versions of node to use
 language: node_js
 node_js:
-- 8.2.1
+- 6
```

Comment:
Since [package.json](https://github.com/fossasia/susper.com/blob/development/package.json#LC83) using `8.2.1` and package.json is used to deploy on heroku then travis should also use `8.2.1` otherwise it loses the whole point of testing the PRs using Travis.